### PR TITLE
Add methods to BoundContract & abigen wrappers

### PR DESCRIPTION
**Pull request by mcortesi**
### Description

New methods to parse event Logs, and expose the ABI object

### Tested

In use in rosetta

### CODE REVIEW

File: accounts/abi/bind/base.go [hunk 1]
```diff
@@ -261,6 +266,19 @@ func (c *BoundContract) transact(opts *TransactOpts, contract *common.Address, i
 	return signedTx, nil
 }
 
+// LogEventName will verify is log belongs to the contract and return the eventName for it
```
Comment (kevjue) [inline accounts/abi/bind/base.go:269]: nit: `verify is log` -> `verify the log`

File: accounts/abi/bind/template.go [hunk 2]
```diff
@@ -351,6 +360,24 @@ var (
 		}
 	{{end}}
 
+	// TryParseLog attemps to parse a log. Returns the parsed log, evenName and wether it was succesfull
```
Comment (kevjue) [inline accounts/abi/bind/template.go:363]: nit:  typo `wether`

File: accounts/abi/bind/base.go (hunk 1 above)
Comment (mcortesi) [reply accounts/abi/bind/base.go]: thanks! Solved

File: accounts/abi/bind/template.go (hunk 2 above)
Comment (mcortesi) [reply accounts/abi/bind/template.go]: thanks! Solved

**Pull request merged**